docs/pvc_scenario.md

```diff
@@ -0,0 +1,21 @@
+### PVC scenario
+Scenario to fill up a given PersistenVolumeClaim by creating a temp file of a given size from a pod that has the volume mounted in the given path. The purpose of this scenario is to fill up a volume to understand faults cause by the application using this volume. 
+
+##### Sample scenario config
+```
+pvc_scenario:                        # Scenario to stop all the nodes for specified duration and restart the nodes
+  pod_name: pod_name                 # Name of the pod with the PVC linked to
```

Comment:
Maybe leverage pod labels to grab the pod-name to ease the CI/automated use cases where we might not know the pod name especially when it's appended with a random id each time it gets created. Something along the lines of user providing the pod-label to query and the number of pods to pick for injecting the chaos. Thoughts?